ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/util/SamplerSCollectionFunctions.scala

```diff
@@ -112,6 +120,9 @@ object SamplerSCollectionFunctions {
   private[samplers] def uniformParams[T: ClassTag, U: ClassTag](s: SCollection[T],
                                                                 keyFn: T => U,
                                                                 prob: Double)
+                                                               (implicit coder0: Coder[T],
```

Comment:
I think these implicits could be simplified similar to how Scio handles keyed SCollections :)  https://github.com/spotify/scio/blob/master/scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala#L461

Reply:
thanks you @ClaireMcGinty! would you mind reviewing the last commit in this PR, "use context bounds to simplify coders"?

Reply:
I'm about to merge this but I can incorporate any suggestions later :)